src/db/service/ConversationService.ts

```diff
@@ -0,0 +1,54 @@
+import ConversationModel, { Conversation } from "../models/conversation.model";
+import { FilterQuery, QueryOptions, UpdateQuery } from "mongoose";
+import { messages } from "../messages";
+
+export default class ConversationService {
+	async createConversation(members: Array<string>) {
+		const conversation = await ConversationModel.create({ members: members });
```

Comment:
should we have `try..catch` here to handle error/failure? Unless it will never fail

Reply:
added